### PR TITLE
Add new questions to FAQ

### PR DESCRIPTION
**Pull request by salarson**

### CODE REVIEW

File: Teams/cloud-recording.md [hunk 1]
```diff
@@ -291,7 +291,19 @@ The expiration date is calculated as the day the meeting recording is created pl
...
 
 The maximum expiration days setting an Admin can apply is 99,999 days, or 273 years. Admins cannot change the expiration date on existing TMRs already uploaded to OneDrive or SharePoint before this feature was released. This protects the intent of the user that owns the TMR. 
+  
+**Does playing the recording change the expiration date?**
+
+No, playback does not impact the expiration date.
+  
+**What happens to the expiration date if the TMR is downloaded and re-uploaded?**
 
+The expiration date will be cleared up re-upload, regardless of the user’s SKU.
```
Comment (brguss) [inline Teams/cloud-recording.md:301]: Small typo: change "cleared up" to "cleared upon"

File: Teams/cloud-recording.md [hunk 2]
```diff
@@ -291,7 +291,19 @@ The expiration date is calculated as the day the meeting recording is created pl
...
+  
+**What happens if I copy or the TMR to a different location or site?**
```
Comment (brguss) [inline Teams/cloud-recording.md:303]: change "copy or" to "copy or move"

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/cloud-recording.md (hunk 1 above)
Comment (JohanFreelancer9) [inline Teams/cloud-recording.md:301]:
```suggestion
The expiration date will be cleared upon re-upload, regardless of the user’s SKU.
```

File: Teams/cloud-recording.md (hunk 2 above)
Comment (JohanFreelancer9) [inline Teams/cloud-recording.md:303]:
```suggestion
**What happens if I copy or move the TMR to a different location or site?**
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @salarson, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**